src/OrchardCore.Modules/OrchardCore.Users.TimeZone/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,19 @@
+using System.Reflection;
```

Comment:
Do we really need another module? Isn't it fine as a feature in the User one? The only required services are all part of already necessary libraries.

Reply:
Not necessary I agree